guidelines.md

```diff
@@ -101,3 +101,61 @@ As an example, consider minimum and maximum length of the reservation:
 * MinLength `P0Y0M1DT0H0M0S`
 * MaxLength `P0Y0M3DT0H0M0S`
 
+# Mews Certification
+
+Before going live in the production environment all connections must be certified by Mews.
+
+### Pre-Certification
+
+- Sign in to the Mews Connector API demo account. Credentials for this can be found in the [guidelines](https://mews-systems.gitbook.io/connector-api/guidelines#authentication) section.
+- Create a unique access token. The client token will always be `E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D`.
+  - Go to **Main menu** > **Settings** > **Marketplace**.
```

Comment:
I was thinking about it and the description is pretty complicated. How about to create a client on demo environment, name it e.g `Wants to integrate`, with nice logo and description and put it to into new and noteworthy category? It's nice and easy. What do you think?

Reply:
Agreed, I have renamed the client and moved it to Noteworthy.